app/api/auth.py

```diff
@@ -345,6 +348,28 @@ def order_invoices(order_identifier):
         return ForbiddenError({'source': ''}, 'Authentication Required to access Invoice').respond()
 
 
+@ticket_blueprint.route('/events/invoices/<string:invoice_identifier>')
+@jwt_required()
+def event_invoices(invoice_identifier):
+    if not current_user:
+        return ForbiddenError({'source': ''}, 'Authentication Required to access Invoice').respond()
+    try:
+        event_invoice = EventInvoice.query.filter_by(identifier=invoice_identifier).first()
+        event_id = event_invoice.event_id
+    except NoResultFound:
+        return NotFoundError({'source': ''}, 'Event Invoice not found').respond()
+    if not current_user.is_organizer(event_id) or not current_user.is_staff:
```

Comment:
The logic is incorrect here. the whole if statement should be true if user is both not the organizer and not from staff.
Instead of "or" it should be "and"

Reply:
@poush Yeah, you're right. I seem to have overlooked that. I've made the change!